content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,94 @@
+features, based on the criteria on the [Feature Status](https://istio.io/latest/about/feature-stages/) page. Thank you
+to everyone that participated in the survey in helping to make Istio 1.9 be our most successful release to date.
+
+More information will be published in the future detailing the Istio community's plans for 2021 in an upcoming blog.
+
+Here are some highlights for this release:
+
+## Feature promotion
+
+### Virtual machine installation promoted to Beta
+
+With many users needing to expand their service mesh to include virtual machines from existing systems, managing the
+complexity of installation is paramount to user success. As such, virtual Machine support has continued to be a high
+priority for the Istio Community. Since Istio 1.6, the community has made major improvements each release for VM
+Support. As of Istio 1.9, [Virtual Mach Installation](/docs/setup/install/virtual-machine/) has been promoted to Beta.
+
+VM support will continue to be a high priority for us in 2021, and you can expect us to see more feature
+graduation in upcoming releases.
+
+### External Istiod promoted to Beta
+
+As with VMs, management of external clusters is vital as Mesh Admins increasingly need to expand their service meshes
+to include remote clusters for many applications (e.g. global distribution, high availability). By promoting this
+feature to Beta we recognize this as a must for adoption and to promote widespread-usage.
+
+## service-apis promoted to Alpha
+
+Support for [Kubernetes service-apis](https://kubernetes-sigs.github.io/service-apis/) has been an active area of
+development since Istio 1.8. These are marketed as the next generation of the Ingress API, and draw inspiration from
+Istio's Gateway and VirtualService. The Istio community is eager to evolve with Kubernetes and these CRDs in upcoming
+releases.
+
+## Third-party providers in Authorization Policy
+
+Authorization Policy has been changed to allow
+[third-party providers](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-ExtensionProvider)
+using [CUSTOM](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action) actions. This allows
+Mesh Admins to use external providers to check if an incoming HTTP request is authorized. This provider must
+implement the [Envoy ext_authz filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_authz_filter)
+authorization check either using HTTP or gRPC.
+
+### Telemetry v2 Request Classification promoted to Beta
+
+Telemetry v2 will see some major improvements in 2021. To jumpstart the year,
+[Request Classification](/docs/tasks/observability/metrics/classify-metrics/) has been promoted to Beta. Promotion of
+this feature will enable users to better monitor the health and well-being of their service mesh.
+
+# Complete support for out-of-mesh metadata for outgoing traffic
+
+?
+
+## Mirroring of images on gcr.io
```

Comment:
cc @howardjohn Please take a closer look at this section. Improvements using the "insert a suggestion" are welcome.